### PR TITLE
Fix 0.5 deprecations

### PR DESCRIPTION
**Pull request by Keno**
Also split 0.4 and 0.5 precompiles. For now the 0.5 one is just a copy, but I'll regenerate it for 0.5 soon.

### CODE REVIEW

**Comment by timholy** [comment]
Did you get the StackOverflowError when running locally on julia-0.5? Or is there something else going on?

**Comment by Keno** [comment]
Compose is currently triggering all sort of memory corruption. One is https://github.com/JuliaLang/julia/pull/16397. The other one I'm still tracking down.

File: src/property.jl [hunk 1]
```diff
@@ -1,7 +1,9 @@
 
 
 abstract PropertyPrimitive
-
+using Compat
+using Compat: String
```
Comment (tkelman) [inline src/property.jl]: should update minimum Compat version

**Comment by yuyichao** [comment]
Many of these seems to be fixed already. The rest (and more) are included in https://github.com/dcjones/Compose.jl/pull/211

**Comment by ranjanan** [comment]
Is this superseded by #211 ?

**Pull request closed without merging**